commerce_coordinator/apps/lms/signal_handlers.py

```diff
@@ -21,6 +21,7 @@ def fulfill_order_placed_send_enroll_in_course(**kwargs):
         edx_lms_user_id=kwargs['edx_lms_user_id'],
         email_opt_in=kwargs['email_opt_in'],
         order_number=kwargs['order_number'],
+        order_id=kwargs['order_id'],
```

Comment:
From what I understand - the `order_number` coming from the views would be the `order_id` already? See: https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/apps/commercetools/sub_messages/tasks.py#L75

Reply:
I can change it as part of this PR. Any other changes I have to make? @JadeyOlivier

Reply:
As well as update the [task](https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/apps/lms/tasks.py#L90) that [calls](https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/apps/commercetools/sub_messages/tasks.py#L114) the signal to include the new attributes

Reply:
Thanks @MushtaqRossier ! 🙌🏼

Reply:
We need to set/pass `order_id: order.id` in [default_params](https://github.com/edx/commerce-coordinator/blob/c700af2f6b8a080340e9c8d724c440a865172f0d/commerce_coordinator/apps/commercetools/sub_messages/tasks.py#L75) where this signal is emitted from. Otherwise, this signal receiver will error on fetching value from `order_id=kwargs['order_id'],`

Reply:
Yeah I'm setting it as part of this PR @shafqatfarhan

Reply:
> As well as update the [task](https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/apps/lms/tasks.py#L90) that [calls](https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/apps/commercetools/sub_messages/tasks.py#L114) the signal to include the new attributes

I think this change is a part of @nyujacky work in: https://2u-internal.atlassian.net/browse/SONIC-330